ios/Classes/FilePickerPlugin.m

```diff
@@ -484,7 +460,7 @@ -(void)picker:(PHPickerViewController *)picker didFinishPicking:(NSArray<PHPicke
     __block NSError * blockError;
     
     for (NSInteger index = 0; index < results.count; ++index) {
-        [urls addObject:[NSNull null]];
+        [urls addObject:[NSURL URLWithString:@""]];
```

Comment:
The warning came from the fact that the `urls` array did not accept null. Since the first element in the array is replaced, I just added an empty NSURL. (and rather than using `new` I went with `URLWithString`, as the former also generates a warning)